doc/example-spouse.md

```diff
@@ -88,12 +88,17 @@ Then we compile our application, as we must do whenever we change `app.ddlog`:
 deepdive compile
 ```
 
-Finally, we tell DeepDive to execute the steps to load the `articles` table:
+Finally, we tell DeepDive to execute the steps to load the `articles` table using the `input/articles.tsv.sh` script. You must have the [full corpus](http://research.signalmedia.co/newsir16/signal-dataset.html) downloaded.
 
 ```bash
 deepdive do articles
 ```
 
+Alternatively, a sample of 1000 articles can be loaded directly by issuing the following command:
+```bash
+deepdive load articles input/articles-1000.tsv.bz2
```

Comment:
Thanks for the edits!  This is a great!

In fact you need a `deepdive mark done articles` after the `deepdive load` to make it equivalent to `deepdive do articles`.  Could you revise this block as well as the one in quickstart?